cmd/root.go

```diff
@@ -62,7 +112,7 @@ func runCmd(args []string, g Getter, out io.Writer) error {
 
 	if len(repoExcludes) > 0 {
 		sort.Strings(repoExcludes)
-		sugar.Debugf("Excluding repos", "repos", repoExcludes)
+		zap.S().Debugf("Excluding repos", "repos", repoExcludes)
```

Comment:
For posterity: I really don't want to bother passing loggers around throughout code as it feels unnecessary and a philosophical difference of opinion.  So these changes have been made to simplify the logging initialization and usage to be global loggers.